src/components/Popups/PopupHighlightError.tsx

```diff
@@ -54,8 +58,11 @@ export default function PopupHighlightError({ shape }: Props): JSX.Element {
         }),
     };
 
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    useOutsideEvent('mousedown', popupRef.current?.popupRef!, onCancel);
```

Comment:
Doesn't `useOutsideEvent already expect a `RefObject`? Or is this to get at the underlying `HTMLElement`

Reply:
Yeah, it needs the underlying HTMLElement.